luaui/Widgets/dbg_widget_auto_reloader.lua

```diff
@@ -0,0 +1,65 @@
+function widget:GetInfo()
+	return {
+		name = "Widget Auto Reloader",
+		desc = "Auto reloads all widgets that have changed when game is out of focus (and once on return focus)",
```

Comment:
nit: The description is slightly incorrect, in that it mentions focus, while the widget works based only on mouse position.